index.css

```diff
@@ -0,0 +1,99 @@
+html{
+    font-size: 62.5%;
+}
+
+*{
+    padding: 0;
+    margin: 0;
+    color: white;
+    font-family:'Montserrat', sans-serif;;
+}
+
+img{
+    max-inline-size: 100%;
+}
+.hero-brand{
+    width: 20rem;
+    height: 20rem;
+}
+.btn a{
+    text-decoration: none;
+    color: white;
```

Comment:
UI looks good Mohammed. Just a suggestion, try to make variables for colours and use them wherever possible instead of colour names or hex codes directly.

Reply:
thank you for the suggestion raksha